ci/deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
`pmacct` is compiled in several flavours of BSD. Since the `Dockerfile` can be used from these systems, and hopefully sometime the CI will add coverage, please do not move to `bash`.

Reply:
No probl: unaware of that requirement. I will rollback when we are happy to merge https://github.com/pmacct/pmacct/pull/503 , as this will require rebase

Reply:
Keeping sh. See last commit ; resolving